eva/catalog/catalog_type.py

```diff
@@ -22,6 +22,7 @@ class Dimension(IntEnum):
 class TableType(IntEnum):
     STRUCTURED_DATA = auto()
     VIDEO_DATA = auto()
+    IMAGE_DATA = auto()
 
```

Comment:
Do we need a very different approach to store AUDIO_DATA or PDF_DATA?

Reply:
It should be similar